src/robotide/editor/kweditor.py

```diff
@@ -584,8 +584,6 @@ def OnKeyDown(self, event):
                 self._open_cell_editor()
             else:
                 event.Skip()
-        if specialkcode != wx.WXK_RETURN:
```

Comment:
If we remove this block, then in Linux, for keyboards that need AltGr to get { or [, the enclose text would not work. (AltGr-7 -> {, AltGr-8 -> [)

Reply:
Ok, I'll find a fix for that too after #2249 is merged.